lib/src/common.dart

```diff
@@ -15,7 +15,7 @@ class Common {
     }
 
     static String versionString() {
-        return CString.fromPtr(bindings.obx_version_string()).val;
+        return Utf8.fromUtf8(bindings.obx_version_string().cast<Utf8>());
```

Comment:
What do you think about changing the definition of `obx_version_string()` in `bindings.dart` to return `Pointer<Utf8>` instead of `Pointer<Uint8>`? So we can get rid of the cast. There are a few functions returning/taking `Pointer<Uint8>` so I guess we can change all of them.

Reply:
```suggestion
        return Utf8.fromUtf8(bindings.obx_version_string());
```

Reply:
I don't see how, sorry. The `#cast<Utf8>` suggests will be lenient in converting the Function's return type from `Pointer<Uint8>` to `Pointer<Utf8>`, but it isnt. Afaik, the `Utf8` type is a struct, and not a scalar like Uint8.

Reply:
I'll try to have a after the merge if what I had in mind works